pkgs/development/python-modules/block-io/default.nix

```diff
@@ -18,9 +19,11 @@ buildPythonPackage rec {
     setuptools
   ];
 
+
   preConfigure = ''
     substituteInPlace setup.py \
-      --replace "ecdsa==0.13" "ecdsa>=0.13"
+      --replace "ecdsa==0.13" "ecdsa>=0.13" \
+      --replace "base58==1.0.3" "base58>=1.0.3"
   '';
```

Comment:
If we're going to be playing with a package's requirements with the tests disabled, could you add a `pythonImportsCheck` so we can ensure it even imports cleanly?

Reply:
good idea, I have done that